script/toolforge-setup

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/bash
```

Comment:
It doesn't really matter since this is only being sourced from other scripts, but it's a bit strange that the shebang line here is a fixed path, whereas the other scripts use `/usr/bin/env`

Reply:
Ah yeah I guess it is a bit inconsistent, I just put this line in so vim would do syntax highlighting in this file 😄